internal/client/githubclient/githubclient_test.go

```diff
@@ -437,11 +437,11 @@ func Test_githubClient_GetWorkflow(t *testing.T) {
 				return
 			}
 			if len(got) != len(test.want) {
-				t.Fatalf("GetWorkflow got %v, want %v", got, test.want)
+				t.Fatalf("GetWorkflow() got = %v, want %v", err, test.want)
 			}
-			for i := 0; i < len(got); i++ {
+			for i := range got {
 				if got[i].Name != test.want[i].Name {
-					t.Fatalf("GetWorkflow got %v, want %v", got[i].Name, test.want[i].Name)
+					t.Errorf("GetWorkflow() got = %v, want %v", got, test.want)
```

Comment:
why this change from `Fatalf` to `Errorf`?

Reply:
Also forgot to change this during the rebase.